CMakeLists.txt

```diff
@@ -536,7 +537,13 @@ endif()
 #
 add_subdirectory(Externals/Bochs_disasm)
 add_subdirectory(Externals/cpp-optparse)
-add_subdirectory(Externals/glslang)
+if(NOT APPLE)
```

Comment:
I don't think this is correct, to only enable Vulkan on Mac. This kind of force disabling is only used when the option definitely won't work.

Reply:
It's already the way it's done in the current build system.
Vulkan is always compiled when not on an apple system.

Reply:
Ah sorry, I get it the test has to be if(APPLE), not if(NOT APPLE)

Reply:
Seems the option definitely won't work on apple.